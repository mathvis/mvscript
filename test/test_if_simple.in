if (a > 0) b;
