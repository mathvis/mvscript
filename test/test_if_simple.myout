Decl (IfBlock (Operation (GreaterThan (VarIdentifier "a") (Type (Int 0)))) (Expr (VarIdentifier "b")) Nothing)
