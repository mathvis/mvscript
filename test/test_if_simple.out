Decl (IfBlock (Operation (GreaterThan (VarIdentifier "a") (0))) (VarIdentifier "b") Nothing)
